Library/Homebrew/brew.sh

```diff
@@ -6,16 +6,13 @@ case "$HOMEBREW_SYSTEM" in
 esac
 
 # Force UTF-8 to avoid encoding issues for users with broken locale settings.
-if ! command -v locale &>/dev/null
-then
-  export LC_ALL=C
-elif [[ "$(locale charmap 2>/dev/null)" != "UTF-8" ]]
+if [[ "$(locale charmap 2>/dev/null)" != "UTF-8" ]]
 then
   if [[ -n "$HOMEBREW_MACOS" ]]
   then
     export LC_ALL="en_US.UTF-8"
   else
-    locales=$(locale -a)
+    locales=$(locale -a 2>/dev/null)
```

Comment:
I'm not a fan of suppressing all error messages from `locale` in this way.

Reply:
so, do you suggest we check `command -v locale` and remove 2>/dev/null ?

Reply:
I suggest reverting to commit bab43d4cc8b828736a91565e5787dbbc5064ea7c. It was good.

Reply:
> I suggest reverting to commit [bab43d4](https://github.com/Homebrew/brew/commit/bab43d4cc8b828736a91565e5787dbbc5064ea7c). It was good.

done